Web/ProductHunt/producthunt-posts-votes.1h.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/local/bin/node
+
```

Comment:
Hey @softvar - do you want to add some metadata please, check out https://github.com/matryer/bitbar#metadata for more. You'll need to at least add the title, version, description & image